packages/disploy/src/structs/CommandInteraction.ts

```diff
@@ -12,7 +12,10 @@ export class CommandInteraction extends BaseInteraction {
 	 * The name of the command.
 	 */
 	public commandName: string;
-
+	/**
+	 * The raw data of the interaction.
+	 */
+	declare public raw: APIApplicationCommandInteraction
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.